src/table/table.scss

```diff
@@ -129,11 +129,18 @@
   display: flex;
   flex-direction: column;
   flex-grow: 1;
-
   @include themed {
     background-color: t(iui-color-background-1);
   }
 
+  &.iui-scroll-snapping {
+    scroll-snap-type: y mandatory;
+
+    .iui-row {
+      scroll-snap-align: start;
```

Comment:
It might be ok to leave this rule in the general `.iui-row` block. It shouldn't have any effect when scroll-snapping is turned off